### PR TITLE
added README.md file explaining how to get DataWarrior file

### PR DESCRIPTION
**Pull request by sonjaleo**
##Description
Create a README.md file to explain how to get the DataWarrior file

##Todos
- [x] Create README.md
- [x]  when README.md okay -> delete instructions in nb 1_3

## Status
- [x] Ready to go

### CODE REVIEW

File: data/filters/DataWarrior/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Use DataWarrior to create Enamine Building Block file
+
+## Table of contents
+1. Open new notebook in notebooks/custom-kinfraglib folder
+2. Load fragment library
+3. save fragment library for DataWarrior
+4. Install DataWarrior
+5. Get Enamine Building Blocks
```
Comment (dominiquesydow) [inline data/filters/DataWarrior/README.md]:
```suggestion
5. Get the Enamine Building Blocks
```

File: data/filters/DataWarrior/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Use DataWarrior to create Enamine Building Block file
```
Comment (dominiquesydow) [inline data/filters/DataWarrior/README.md]:
```suggestion
# Use DataWarrior to create a Enamine Building Block file
```

File: data/filters/DataWarrior/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## Table of contents
+1. Open new notebook in notebooks/custom-kinfraglib folder
+2. Load fragment library
+3. save fragment library for DataWarrior
```
Comment (dominiquesydow) [inline data/filters/DataWarrior/README.md]: I would summarize these three steps under the headline "1. Prepare KinFragLib fragment library for DataWarrior"

File: data/filters/DataWarrior/README.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+4. Install DataWarrior
+5. Get Enamine Building Blocks
+
+## 1. Open new notebook in notebooks/custom-kinfraglib folder
```
Comment (dominiquesydow) [inline data/filters/DataWarrior/README.md]:
```suggestion

## 1. Prepare KinFragLib fragment library for DataWarrior

### a. Open a new notebook in the `notebooks/custom-kinfraglib` folder
```

File: data/filters/DataWarrior/README.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+## 2. Load fragment library
```
Comment (dominiquesydow) [inline data/filters/DataWarrior/README.md]:
```suggestion
## b. Load the fragment library
```

File: data/filters/DataWarrior/README.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+PATH_DATA = HERE / '../../data'
+```
+
+```python
+fragment_library = utils.read_fragment_library(PATH_DATA / 'fragment_library')
+```
+
+```python
+pd.concat(fragment_library).reset_index(drop=True).shape
+```
```
Comment (dominiquesydow) [inline data/filters/DataWarrior/README.md:38]: Maybe write all code in a single cell? Would be easier to copy in one go.

File: data/filters/DataWarrior/README.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+Apply pre-filters
+- removing fragments in pool X
+- removing duplicates
+- removing fragments without dummy atoms (unfragmented ligands)
+- removing fragments only connecting to pool X
```
Comment (dominiquesydow) [inline data/filters/DataWarrior/README.md:44]:
```suggestion
Apply pre-filters that remove
- fragments in pool X
- duplicates
- fragments without dummy atoms (unfragmented ligands)
- fragments only connecting to pool X
```

File: data/filters/DataWarrior/README.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+```python
+fragment_library_pre_filtered = filters.prefilters.pre_filters(
+    fragment_library)
```
Comment (dominiquesydow) [inline data/filters/DataWarrior/README.md:50]:
```suggestion
fragment_library_pre_filtered = filters.prefilters.pre_filters(
    fragment_library
)
```

File: data/filters/DataWarrior/README.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+## 3. save fragment library for DataWarrior
```
Comment (dominiquesydow) [inline data/filters/DataWarrior/README.md]:
```suggestion
## c. Save the fragment library to be processed with DataWarrior
```

File: data/filters/DataWarrior/README.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+
+## 4. Install DataWarrior
+Go to the [DataWarrior](https://openmolecules.org/datawarrior/download.html) web page and install 
+DataWarrior for your OS
+
+## 5. Get Enamine Building Blocks
+- open DataWarrior
+- open file 'smiles_wo_dummy.sdf' in the fragment_library directory
```
Comment (dominiquesydow) [inline data/filters/DataWarrior/README.md]:
```suggestion
- open file `smiles_wo_dummy.sdf` in the `data/fragment_library` directory 
```

File: data/filters/DataWarrior/README.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+- open DataWarrior
+- open file 'smiles_wo_dummy.sdf' in the fragment_library directory 
+  * choose 'yes' when DataWarrior asks if it should interpret stereo centers as absolute
+- mark all structures 
+  * go to Database 
+  * Search Enamine Building Blocks
+  * choose superstructures at the top dropdown menu
+  * set maximum row count to 100000 
+  * set minimun package size to 1 
+  * choose " retrieve cheapest..." at the bottom
```
Comment (dominiquesydow) [inline data/filters/DataWarrior/README.md]:
```suggestion
  * choose "retrieve cheapest..." at the bottom 
```

File: data/filters/DataWarrior/README.md [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+
+
+## 4. Install DataWarrior
+Go to the [DataWarrior](https://openmolecules.org/datawarrior/download.html) web page and install 
+DataWarrior for your OS
+
+## 5. Get Enamine Building Blocks
+- open DataWarrior
+- open file 'smiles_wo_dummy.sdf' in the fragment_library directory 
+  * choose 'yes' when DataWarrior asks if it should interpret stereo centers as absolute
```
Comment (dominiquesydow) [inline data/filters/DataWarrior/README.md]:
```suggestion
  * choose "yes" when DataWarrior asks if it should interpret stereo centers as absolute
```

File: data/filters/DataWarrior/README.md [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+  * Search Enamine Building Blocks
+  * choose superstructures at the top dropdown menu
+  * set maximum row count to 100000 
+  * set minimun package size to 1 
+  * choose " retrieve cheapest..." at the bottom 
+  * run query
+- go to file dropdown menu 
+  * save special 
+  * choose SDfile 
+  * name it "Enamine_Building_Blocks.sdf" and save it in a folder called DataWarrior in same
```
Comment (dominiquesydow) [inline data/filters/DataWarrior/README.md]:
```suggestion
  * name it `Enamine_Building_Blocks.sdf` and save it in a folder called DataWarrior in same 
```

**Review by dominiquesydow — COMMENTED** [review]
Hi @sonjaleo, 

Great README with all the details needed, well done! I suggest a few format changes but that's it. 

You could link to your notebook as an example if you want.

File: data/filters/DataWarrior/README.md (hunk 3 above)
Comment (sonjaleo) [reply data/filters/DataWarrior/README.md]: Good idea :)

**Comment by sonjaleo** [comment]
> Hi @sonjaleo,
> 
> Great README with all the details needed, well done! I suggest a few format changes but that's it.
> 
> You could link to your notebook as an example if you want.

@dominiquesydow not sure which notebook you mean? Because the steps that are now in the README should be deleted in the notebook. Or do you mean linking the notebook where the DataWarrior file is used?

**Comment by dominiquesydow** [comment]
> @dominiquesydow not sure which notebook you mean? Because the steps that are now in the README should be deleted in the notebook. Or do you mean linking the notebook where the DataWarrior file is used?

Good point... you do have a notebook or script that you use to prepare the input fragments for DataWarrior, right? I would probably write something like

> Please find the notebook used to prepare the KinFragLib fragment library for the DataWarrior input at `notebooks/XXX.ipynb`.

**Comment by sonjaleo** [comment]
> > @dominiquesydow not sure which notebook you mean? Because the steps that are now in the README should be deleted in the notebook. Or do you mean linking the notebook where the DataWarrior file is used?
> 
> Good point... you do have a notebook or script that you use to prepare the input fragments for DataWarrior, right? I would probably write something like
> 
> > Please find the notebook used to prepare the KinFragLib fragment library for the DataWarrior input at `notebooks/XXX.ipynb`.

I wrote a function for the prefiltering which is done in every notebook. The only thing (which I would delete from ne notebook now that it is in the readme) is to safe the smiles without dummy atoms in a file.

I would think the only notebook I could mention is something like
"The created DataWarrior file is used in `notebooks/custom_kinfraglib/1_3_custom_filters_synthesizability.ipynb`"

Edit: I could add a link to notebooks/kinfraglib/3_1_fragment_library_reduced.ipynb where the prefiltering is explained?

**Comment by dominiquesydow** [comment]
> I would think the only notebook I could mention is something like
"The created DataWarrior file is used in notebooks/custom_kinfraglib/1_3_custom_filters_synthesizability.ipynb"

> Edit: I could add a link to notebooks/kinfraglib/3_1_fragment_library_reduced.ipynb where the prefiltering is explained?

Perfect! Yes to both; then the user can find more information on (a) the pre-filtering [before DataWarrior] and (b) what can be done with the new dataset [after DataWarrior].

**Pull request merged**